.github/workflows/fpb-lint.yml

```diff
@@ -2,6 +2,11 @@ name: free-programming-books-lint
 
 on: [push, pull_request]
 
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GITHUB_RUN_ID: ${{ github.run_id }}
+  PR: ${{ github.event.pull_request.html_url }}
+
 jobs:
   build:
```

Comment:
About current workflow error at `Print output` task.... 

![image](https://user-images.githubusercontent.com/3125580/178809154-e48a3171-5457-455d-aec7-3e3c8070d298.png)

Reading https://www.damirscorner.com/blog/posts/20220506-TokenPermissionsForGithubActions.html... a bit I think that is about GitHub token permissions

See Seth workarround at PR #5557 : https://github.com/SethFalco/free-programming-books/blob/e9f7dff3be87e640a72f97bd4eec1be79ec2b43a/.github/workflows/fpb-lint.yml#L7-L9

Reply:
Note, my workaround is insecure and shouldn't be used if I understand correctly.

The real solution is in my comment below. (Split into 2 separate workflows, one creates the message, the other reads the message and posts the review.)